### PR TITLE
enhacements to pyswmm.Simulation class

### PR DESCRIPTION
**Pull request by bemcdonnell**
Some more simulation class features have been added

### CODE REVIEW

**Comment by bemcdonnell** [comment]
This covers #27

**Comment by bemcdonnell** [comment]
Also updated the README today.

File: pyswmm/pyswmm.py [hunk 1]
```diff
@@ -166,5 +166,103 @@ def execute(self):
         >>> swmm_model.swmm_open()
         >>> swmm_model.swmmExec()     
         """
-        
         self._model.swmmExec()
+        
+    @property
+    def starttime(self):
```
Comment (goanpeca) [inline pyswmm/pyswmm.py]: start_time ?

File: pyswmm/pyswmm.py [hunk 2]
```diff
@@ -166,5 +166,103 @@ def execute(self):
...
+        """
+        return self._model.getSimulationDateTime(SimulationTime.StartDateTime)
+    
+    @starttime.setter
+    def starttime(self):
+        """Set simulation Start time"""
+        self._model.setSimulationDateTime(SimulationTime.StartDateTime, dtimeval)
+
+    @property
+    def endtime(self):
```
Comment (goanpeca) [inline pyswmm/pyswmm.py]: end_time?

File: pyswmm/pyswmm.py [hunk 3]
```diff
@@ -166,5 +166,103 @@ def execute(self):
...
+        """
+        return self._model.getSimulationDateTime(SimulationTime.EndDateTime)
+    
+    @endtime.setter
+    def endtime(self):
+        """Set simulation End time"""
+        self._model.setSimulationDateTime(SimulationTime.EndDateTime, dtimeval)
+
+    @property
+    def reportstart(self):
```
Comment (goanpeca) [inline pyswmm/pyswmm.py]: report_start ?

File: pyswmm/pyswmm.py [hunk 4]
```diff
@@ -166,5 +166,103 @@ def execute(self):
...
+        """
+        return self._model.getSimulationDateTime(SimulationTime.ReportStart)
+    
+    @reportstart.setter
+    def reportstart(self):
+        """Set simulation report start time"""
+        self._model.setSimulationDateTime(SimulationTime.ReportStart, dtimeval)
+
+    @property
+    def flowunits(self):
```
Comment (goanpeca) [inline pyswmm/pyswmm.py]: flow_units ?

File: pyswmm/pyswmm.py [hunk 5]
```diff
@@ -166,5 +166,103 @@ def execute(self):
...
+        >>>
+        >>> with Simulation('../test/TestModel1_weirSetting.inp') as sim:
+        ...     print sim.flowunits
+        >>>
+        >>> CFS
+        """
+        return self._model.getSimUnit(SimulationUnits.FlowUnits)
+
+    @property
+    def systemunits(self):
```
Comment (goanpeca) [inline pyswmm/pyswmm.py]: system_units?

**Pull request merged**